src/DetailsView/components/tab-stops/failed-instance-panel.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { ActionAndCancelButtonsComponent } from 'DetailsView/components/action-and-cancel-buttons-component';
+import { GenericPanel, GenericPanelProps } from 'DetailsView/components/generic-panel';
+import { TextField } from 'office-ui-fabric-react';
+import * as React from 'react';
+import * as styles from '../failure-instance-panel.scss';
+
+export interface FailedInstancePanelProps {
+    isOpen: boolean;
+    instanceDescription: string;
+    headerText: string;
+    confirmButtonText: string;
+    onConfirm: () => void;
+    onChange: (event: React.SyntheticEvent, description: string) => void;
+    onDismiss: () => void;
+}
+
+export class FailedInstancePanel extends React.Component<FailedInstancePanelProps> {
```

Comment:
curiosity question, how come this is extending react.component as opposed to being a namedFC?

Reply:
Good catch! I'll get to it in another PR (so I can complete this now and throw up the wiring PR immediately).

This was from when initially the component had state but eventually was removed due to design changes.